dev/io.openliberty.jakarta.pages.3.1/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,276 @@
+#Ant properties
+#Automatically generated by the ant prepare.settings.files task
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.compliance=1.8
+org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+org.eclipse.jdt.core.compiler.source=1.8
```

Comment:
Shouldn't these 1.8 references be 11? Should be able to validate against the Servlet 6.0 bundle you're working on.